regions/io/crtf/read.py

```diff
@@ -29,7 +29,7 @@
 regex_length = re.compile(r'(?:\[[^=]*\])+[,]\s*([\w.+-]*)\]')
 
 # Extracts each 'parameter=value' pair.
-regex_meta = re.compile(r'((?:\w+\s*=\s*[^,\[\]]+)|(?:\w+\s*=\s*\[.*?\]))')
+regex_meta = re.compile(r'(?:(\w+)\s*=[\s\'\"]*([^,\[\]]+?)[\'\",]+)|(?:(\w+)\s*=\s*\[(.*?)\])')
```

Comment:
if at all possible, add a comment describing this regex and/or indicating some strings it should match.  These should also be included in tests, of course, but regex's are really hard to read in general

Reply:
ya.. I will try to convert them into `VERBOSE` mode (there is a option to describe what you are doing in the regex)